packages/react-core/src/components/SimpleList/examples/SimpleListUncontrolled.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { SimpleList, SimpleListItem } from '@patternfly/react-core';
+
+export const SimpleListUncontrolledDemo: React.FunctionComponent = () => {
+  const [activeItem, setActiveItem] = React.useState(0);
+
+  const onSelect = (selectedItem: any, selectedItemProps: any) => {
```

Comment:
can we update the props' types in this `onSelect` handler to match the `SimpleList` prop types? 
```
onSelect?: (
    ref: React.RefObject<HTMLButtonElement> | React.RefObject<HTMLAnchorElement>,
    props: SimpleListItemProps
  ) => void;
```

Reply:
agree, did more work to match this request